core/services/workflows/metering/metering.go

```diff
@@ -99,7 +102,11 @@ func (r *Report) Reserve(ctx context.Context) error {
 	defer r.mu.Unlock()
 
 	if r.client == nil {
-		return ErrNoBillingClient
+		r.lggr.Error("billing client not set; switching to metering mode")
+		r.balance.meteringMode = true
+		r.ready = true
```

Comment:
This line is repeated 4 times. Could it be moved to the top so that it's only used once?